src/server/services/BruincastServices.js

```diff
@@ -166,6 +167,17 @@ class BruincastServices {
         delete entry._id;
         delete entry.timestamp;
       }
+
+      // Sort each course's media listings by date. If date is the same, then sort by title.
+      courseMedia.listings.sort(function(listing1, listing2) {
+        if (listing1.date < listing2.date) return -1;
+        if (listing2.date < listing1.date) return 1;
+        if (listing1.date === listing2.date && listing1.title < listing2.title)
```

Comment:
I don't think you need date1===date2? If we reach this line we already have date1===date2 for sure right?

Reply:
Oh yeah, that's true, whoops